GameData/RealismOverhaul/RO_RCS_Fixes.cfg

```diff
@@ -40,3 +32,11 @@
 		}
 	}
 }
+
+@PART[*]:HAS[@MODULE[ModuleRCS]]:AFTER[zzzRealismOverhaul]
+{
+	@MODULE[ModuleRCS]
+	{
+		@name = ModuleRCSFX
+	}
+}
```

Comment:
Prefer the other way around, just editing the HAS block in your patch.  You also want to operate on ModuleRCSFX, or at least ModuleRCS* and not just ModuleRCS.

Reply:
Hm, I'm pretty sure the current patch preserves the semantics of the original patch (not that I understand what it does and why it does it the way it does). 

The original applied only to parts with a `ModuleRCS`, renaming the module and rewriting the effects.

The new patch does the same, but only applies the effect rewriting if the part doesn't already have a `ModuleWaterfallFX`. If we renamed first, then we don't know which of the parts containing `ModuleRCSFX` originally had a `ModuleRCS` anymore.

Reply:
Ok.  I think I wasn't understanding the intent of the original patch [specifically: only rewriting the effects on ModuleRCS].  I'm good with this.